events-api/src/modules/tickets/dto/create-payment.dto.ts

```diff
@@ -0,0 +1,48 @@
+import { IsNotEmpty, IsNumber, IsPositive, IsString, Length } from 'class-validator';
+import { IsDateFormatValid } from '../../../shared/validators/date-format.validator';
+import { TimeFormat } from '../../../shared/constants/time.constant';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class CreatePaymentDto {
+  @ApiProperty({
+    description: 'Last 4 digits of card',
+    example: 1234,
+  })
+  @IsNotEmpty()
+  @IsPositive()
+  @Length(4, 4, { message: 'Must contain exactly 4 digits' })
+  last4Digits: number;
+
+  @ApiProperty({
+    description: 'Expiry date of the card in MM/YY format',
+    example: '12/24',
+  })
+  @IsString()
+  @IsNotEmpty()
+  @IsDateFormatValid(TimeFormat.CalendarForCard)
```

Comment:
Maybe rename the CalendarForCard CardExpiration? I don't see that we are doing something with calendar